docs/volto-light-theme/adding-installing-new-block.md

```diff
@@ -0,0 +1,11 @@
+---
+myst:
+  html_meta:
+    "description": "Adding Installing New Block"
+    "property=og:description": "Adding Installing New Block"
+    "property=og:title": "Adding Installing New Block"
+    "keywords": "Plone, Volto, Training"
+---
+
+
+# Adding Installing New Block
```

Comment:
Either Add or Install, not both. See also that headings should have only the first word capitalized, excluding of course proper nouns.

```suggestion
# Add a new block
```